awsagentprovider/src/test/java/software/amazon/opentelemetry/javaagent/providers/AwsTracerCustomizerProviderTest.java

```diff
@@ -0,0 +1,54 @@
+package software.amazon.opentelemetry.javaagent.providers;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.AfterEach;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+class AwsTracerCustomizerProviderTest {
```

Comment:
Can you add the following two test cases? 

* when OTEL_AWS_APPLICATION_SIGNALS_ENABLED is true, check the relevant properties are set with default
* when OTEL_AWS_APPLICATION_SIGNALS_ENABLED is false, check the relevant properties are not set